proto/nameserver2.proto

```diff
@@ -438,6 +444,7 @@ message OpenFileResponse {
     optional ProtoSession protoSession = 2;
     optional FileInfo   fileInfo = 3;
     optional CloneSourceSegment cloneSourceSegment = 4;
+    optional uint32 permission = 5;
```

Comment:
`permission` might be unnecessary, because an open operation is either succeeded or failed, if it's succeeded, we can deduce its permission from openflags.

Reply:
fix.

Reply:
> fix.

use bool success instead of it.